### ISSUE TITLE
Accessibility Pass

### ISSUE THREAD

**Issue by sigrunixia** [carlonicora/obsidian-rpg-manager#167]
## Scope
Limited to documentation and visual features found on Obsidian-RPG-Manager Github. An accessibility pass will be performed in plugin later during design overhaul.

## Summary
With release of v3.0 imminent, existing documentation will be checked for accessibility access. The existing functionality of GitHub users ability to change accessibility on their end is not greatest, so we will do what we can.  This includes but is not limited to:
- [x] Recoloring labels to conform to datapoints in *[Points of view: Color blindness](https://www.nature.com/articles/nmeth.1618#Fig2)*
- [x] Ensuring clarity of explained concepts to be understood by those who have obtained basic comprehension of English as a Second Language
- [x] #188
- [x] Explore utilization of the [Atkinson Hyperlegible Font](https://brailleinstitute.org/freefont) as it is also a Google Webfont. 

## Realistic Limitations
It is innately understood that this will be an ongoing project and that a Github repository cannot meet the complex standards of every person. Furthermore, there is some level of technical assistance expected on the user end if there are any users who need accessibility help. 

### Assumed To Be the Most Common Populations (Unordered)
- Color blindness
- Visual challenges in text size and shape
- Physical dexterity in the hands 
- Multi-language speakers and readers

## Expected Completion Date
Current expected completion date of this pass is to match Milestone v3.2.

**Comment by sigrunixia** [comment]
These are the two palettes that are created as a result of the recommendations from *[Points of view: Color blindness](https://www.nature.com/articles/nmeth.1618#Fig2)*.

![palette-colorblindess-contrast](https://user-images.githubusercontent.com/68425372/195174569-948d32e7-eddc-4101-9934-6e4c311b5cf7.png)
![nature_colorblindness_palette](https://user-images.githubusercontent.com/68425372/195174577-8cd621ac-51d0-466b-bdf7-566c90d62043.png)

**Comment by carlonicora** [comment]
Hi @sigrunixia 

I think the proposals are absolutely great. I have absolutely no issue in changing colours or any other details on GitHub

**Comment by sigrunixia** [comment]
## 2022-10-12
- Wontfix - Color changed to cream
- Wontfix description changed to "This request will not be worked on."
- Documentation - Color changed to Offwhite
- Documentation description changed to "Documentation may need to be amended or created."
- Feature - Color changed to Vermillion
- Feature description changed to "This topic is related to a new feature."
- Bug - Color Changed to Yellow
- Bug description changed to " Something is not working and will need to be fixed."
- All User Feedback Requested changed to All Feedback Requested
- All Feedback Requested description changed to "The developer has questions and wants feedback from users."
- All Feedback Requested - Color Changed to Sky Blue
- Refactoring - Color changed to Orange
- Refactoring description updated to: "This topic or pull request includes operational improvements to the plugin."
- No Bug renamed to Q&A
- Q&A given the description of "Questions for developers, collaborators, and contributors"
- Duplicate description changed to "This topic or pull request is a duplicate."
- Patch description changed to "This topic is related to an upgrade to an existing feature."

Additional colors still needed for "Ready", "Question", "Patch", "No Bug", and "Duplicate" which I will get to in my task above. 

## Proposed Changes
"Question" would be changed to "Discovery"
Discovery Description changed to: Additional information is needed before proceeding on this topic.

**Comment by sigrunixia** [comment]
Closing as the remainder can easily be handled in #174 and #168. :)
